README.md

```diff
@@ -478,7 +502,29 @@ func loadHandler(completion: @escaping ([Meal]?, RequestError?) -> Void ) {
 ```
 
 Now when you perform a `GET` call to your server it will retrieve the meals from your database.
-You can verify this by going to [http://localhost:8080/meals](http://localhost:8080/meals), where you should see your meals.
+
+Update the summaryHandler function to get the meals from the database:
+1.  Inside the `summaryHandler` function add the following line:
+```swift
+Meal.findAll { meals, error in
+    if let meals = meals {
+        completion(Summary(meals), nil)
+    }
+}
+```
+
+2. Your completed `summaryHandler` function should now look as follows:
+```swift
+func summaryHandler(completion: @escaping ([Summary]?, RequestError?) -> Void ) {
+    Meal.findAll { meals, error in
+        if let meals = meals {
+            completion(Summary(meals), nil)
```

Comment:
This doesn't have error handling, and will hang forever if `meals` doesn't resolve properly.